app/Http/Controllers/UserListController.php

```diff
@@ -8,11 +8,11 @@
 
 class UserListController extends Controller
 {
-    public function create(){
+    public function create() {
         return view('users.lists.create');
     }
 
-    public function store(Request $request){
+    public function store(Request $request) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->